core/pom.xml

```diff
@@ -178,6 +177,12 @@
 	</build>
 	<dependencies>
 		<dependency>
+			<groupId>org.yaml</groupId>
+			<artifactId>snakeyaml</artifactId>
+			<version>1.15</version>
```

Comment:
This is the version that used to be shaded by jackson: https://github.com/FasterXML/jackson-dataformat-yaml/blob/c69224917fd235732e51c8327f23af7f7038108f/pom.xml#L56.